option.rb

```diff
@@ -0,0 +1,27 @@
+require './retrieve_data'
+require './store_data'
+
+class Option
+  def self.option(option, app) # rubocop:disable Metrics/CyclomaticComplexity
+    retrieve_data(app)
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, there's a problem with having this `retrieve_data` method here:

Every time I select an option, and the loop starts again, data gets duplicated over and over:

![image](https://github.com/Zohra-Neda/OOP-School-Library/assets/77069011/baeb300e-b3cb-4905-b34a-f8cac68a4af5)

To fix this, I suggest calling this method only when the app starts. 🤓 🔧